src/r_data.c

```diff
@@ -1143,7 +1143,11 @@ int R_FlatNumForName(const char *name)    // killough -- const added
   if (i == NO_TEXTURE)
   {
     I_Printf(VB_WARNING, "R_FlatNumForName: %.8s not found", name);
-    return i;
+    i = (W_CheckNumForName)("-NO_TEX-", ns_flats); // highlight missing flats
+    if (i == NO_TEXTURE)
```

Comment:
Do we check for the existence of lumps that we provide ourselves in the data file?